### PR TITLE
Fix issue 21552: Encoding cp866 ru

### PR DESCRIPTION
**Pull request by yurivrn**
I needed this encoding for my project at work. I made it and decided to post it here in case someone else finds it useful. If there is another way to add an encoding without changing Phobos (like creating an instance of an abstract class) I could try doing that too. But I really could use a piece of advice here. I failed to find anything relevant to my problem in documentation. That's why I'd be thankful for your feedback :)

### CODE REVIEW

File: std/encoding.d [hunk 1]
```diff
@@ -2481,7 +2589,8 @@ do
         string asciiCharString = to!string(iota(0, 128, 1));
 
         alias Types = AliasSeq!(string, Latin1String, Latin2String, AsciiString,
-            Windows1250String, Windows1251String, Windows1252String, dstring, wstring);
+            Cp866String, Windows1250String, Windows1251String, Windows1252String,
```
Comment (thewilsonator) [inline std/encoding.d:2592]:
```
Check for trailing whitespace
  | grep -nr '[[:blank:]]$' $(find etc std -name '*.d'); test $? -eq 1
  | std/encoding.d:2592:            Cp866String, Windows1250String, Windows1251String, Windows1252String,
  | posix.mak:581: recipe for target 'style_lint_shellcmds' failed
```

**Review by thewilsonator — APPROVED** [review]
Please create and reference a bugzilla and title  the commit _message_ "Fix issue #####: description of issue" so that it is picked in the release notes. Alternately,  please add a changelog  entry.

File: std/encoding.d (hunk 1 above)
Comment (yurivrn) [reply std/encoding.d]: Done. Fixed it

**Comment by RazvanN7** [comment]
@yurivrn Please address @thewilsonator 's comments [1] so that we can merge this.

[1] > Please create and reference a bugzilla and title the commit message "Fix issue #####: description of issue" so that it is picked in the release notes. Alternately, please add a changelog entry.

**Comment by yurivrn** [comment]
Ok, got it, I'll do that as soon as I can

> Please create and reference a bugzilla and title the commit _message_ "Fix issue #####: description of issue" so that it is picked in the release notes. Alternately, please add a changelog entry.

**Comment by RazvanN7** [comment]
@yurivrn The github bot can automatically reference bugzilla issues (and close them when the PRs are merged) provided that  there is a commit message that contains the message "Fix Issue $Insert_issue_number  - $Insert issue description". I see that you have changed the PR title, unfortunately, that will not do.

What this PR still requires is:
- a changelog entry (please see the changelog directory in the root phobos repo; it contains a readme on how to describe the added feature)
- a commit that links the bugzilla issue.

Thanks!

**Comment by RazvanN7** [comment]
@yurivrn You still have not pushed a commit that contains "Fix Issue 21522" in the commit message

**Comment by RazvanN7** [comment]
Ping @yurivrn

**Comment by RazvanN7** [comment]
Author seems to be gone. I have posted a link to this PR in the issue. Closing for now.

**Pull request closed without merging**